graylog2-server/src/test/java/org/graylog2/database/MongoCollectionsTest.java

```diff
@@ -56,12 +54,12 @@ class MongoCollectionsTest {
     private MongoCollections collections;
     private EncryptedValueService encryptedValueService;
 
-    record Person(@JsonProperty("id") @Id @org.mongojack.ObjectId String id,
+    record Person(String id,
```

Comment:
Same as above. Please check if annotation inheritance works for records. Thanks!

Reply:
The record behaves correctly but it's not very intuitive why it does so. I brought the annotations back to make it more explicit.